test/e2e/clusterplatform_test.go

```diff
@@ -0,0 +1,159 @@
+				returnedValue, _ := utils.Run(cmd)
+				println(string(returnedValue))
+				return returnedValue
+			}, 20*time.Minute, 5).Should(Equal([]byte("''")))
+
+			By("Evaluate status of SonataFlowClusterPlatform CR")
+			cmd = exec.Command("kubectl", "patch", "SonataFlowClusterPlatform", "cluster", "--type", "merge", "-p", `{"spec": {"platformRef": {"namespace": "`+targetNamespace+`"}}}`)
+			_, err = utils.Run(cmd)
+			Expect(err).NotTo(HaveOccurred())
+			EventuallyWithOffset(1, func() error {
+				cmd = exec.Command("kubectl", "wait", "SonataFlowClusterPlatform", "cluster", "--for", "condition=Succeed", "--timeout=5s")
+				_, err = utils.Run(cmd)
+				return err
+			}, 20*time.Minute, 5).Should(Succeed())
+
+			EventuallyWithOffset(1, func() error {
+				var err error
+				cmd := exec.Command("kubectl", "kustomize", filepath.Join(projectDir,
+					testcaseDir, profile, persistenceType))
+				manifests, err = utils.Run(cmd)
+				return err
+			}, time.Minute, time.Second).Should(Succeed())
+
+			cmd = exec.Command("kubectl", "create", "-n", targetNamespace2, "-f", "-")
+			cmd.Stdin = bytes.NewBuffer(manifests)
+			_, err = utils.Run(cmd)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Wait for SonatatFlowPlatform CR in " + targetNamespace2 + " to be ready")
+			dataIndexServiceUrl := services.GenerateServiceURL(constants.KogitoServiceURLProtocol, targetNamespace, "sonataflow-platform-"+constants.DataIndexServiceName)
+			jobServiceUrl := services.GenerateServiceURL(constants.KogitoServiceURLProtocol, targetNamespace, "sonataflow-platform-"+constants.JobServiceName)
+			// wait for platform to be ready
+			EventuallyWithOffset(1, func() error {
+				cmd = exec.Command("kubectl", "wait", "sfplatform", "-n", targetNamespace2, "sonataflow-platform", "--for", "condition=Succeed", "--timeout=5s")
+				_, err = utils.Run(cmd)
+				return err
+			}, 20*time.Minute, 5).Should(Succeed())
+			EventuallyWithOffset(1, func() []byte {
+				cmd = exec.Command("kubectl", "get", "sfplatform", "-n", targetNamespace2, "sonataflow-platform", "-o", "jsonpath='{.status.clusterPlatformRef.services.dataIndexRef.url}'")
+				returnedValue, _ := utils.Run(cmd)
+				return returnedValue
+			}, 20*time.Minute, 5).Should(Equal([]byte("'" + dataIndexServiceUrl + "'")))
+			EventuallyWithOffset(1, func() []byte {
+				cmd = exec.Command("kubectl", "get", "sfplatform", "-n", targetNamespace2, "sonataflow-platform", "-o", "jsonpath='{.status.clusterPlatformRef.services.jobServiceRef.url}'")
+				returnedValue, _ := utils.Run(cmd)
+				return returnedValue
+			}, 20*time.Minute, 5).Should(Equal([]byte("'" + jobServiceUrl + "'")))
+
+			cmd = exec.Command("kubectl", "delete", "SonataFlowClusterPlatform", "cluster", "--wait")
+			_, err = utils.Run(cmd)
+			Expect(err).NotTo(HaveOccurred())
+		},
+			Entry("without services configured", test.GetSonataFlowE2EPlatformNoServicesDirectory(), dev, ephemeral),
```

Comment:
Do you think it's worth covering additional use cases, for instance when one of the services is already provided in the target namespace, or when both are?

Reply:
done